src/benchcab/model.py

```diff
@@ -207,12 +205,20 @@ def build(self, modules: list[str], mpi: bool, coverage: bool):
                 )
 
                 self.subprocess_handler.run_cmd(
-                    "cmake -S . -B build " + " ".join(cmake_args), env=env
+                    "cmake -S . -B build -DCABLE_MPI=OFF " + " ".join(cmake_args), env=env
                 )
-                self.subprocess_handler.run_cmd("cmake --build build ", env=env)
+                self.subprocess_handler.run_cmd("cmake --build build", env=env)
                 self.subprocess_handler.run_cmd(
                     "cmake --install build --prefix .", env=env
                 )
+                if mpi:
+                    self.subprocess_handler.run_cmd(
+                        "cmake -S . -B build_mpi -DCABLE_MPI=ON " + " ".join(cmake_args), env=env
+                    )
+                    self.subprocess_handler.run_cmd("cmake --build build_mpi", env=env)
```

Comment:
TODO: it would be cleaner if the `build_mpi` directory was under `./build/build_mpi` and similarly for the serial case so that `build_mpi` stays under the `.gitignore`'d build directory.